qiskit/pulse/transforms/alignments.py

```diff
@@ -46,29 +44,53 @@ def align(self, schedule: Schedule) -> Schedule:
         """
         pass
 
+    @deprecated_functionality
     def to_dict(self) -> Dict[str, Any]:
         """Returns dictionary to represent this alignment."""
         return {"alignment": self.__class__.__name__}
 
+    @deprecated_functionality
+    @property
+    def is_sequential(self):
+        """Deprecated attribute.
+
+        Use superclass :class:`.SequentialContext` or :class:`.ParallelContext`.
```

Comment:
What is the super class for this? `AlignmentKind` just inherits from `ABC`? Should this be `subclass` because `SequentialContext` and `ParallelContext` inherit from `AlignmentKind`